### DISCUSSION TITLE
how download

### DISCUSSION THREAD

**Discussion by UlyssesYYZ** [gitextensions/gitextensions#8370] [General] [answered]
there isn't any buttons to download it

**Comment by vbjay** [comment, marked as answer]
https://github.com/gitextensions/gitextensions

![image](https://user-images.githubusercontent.com/3629489/89108768-22b06080-d409-11ea-9801-6ffdfb08f4e7.png)

**Comment by RussKie** [comment]
Please scroll down the front page
